tests/features/taxonomy_browser.feature

```diff
@@ -0,0 +1,94 @@
+@api
+Feature: Taxonomy browser
+  In order to test ‘Taxonomy browser’ feature
+  As an administrator
+  I want to make sure ‘Taxonomy browser’ allows creating blocks with taxonomy vocabularies
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+    When I visit "/admin/config/taxonomy_browser/settings_en"
+    When I check "edit-taxonomy-browser-vocabulary-2"
+    And  I press "Save configuration"
+    Then I should see the text "The configuration options have been saved."
```

Comment:
We don't need to add the configuration as a background. You can first run the scenario "Administrator user activates the vocabulary on taxonomy browser page and configures the block" and then create the view.